service/controller/v24/templates/cloudformation/cpi/main_data.go

```diff
@@ -0,0 +1,6 @@
+package cpi
+
+// CPI is the adapter collection for the Control Plane Initializer management.
+type CPI struct {
+	IAMRoles *CPIIAMRoles
+}
```

Comment:
I am also not happy with the file names in here. Would be cool if you could suggest something cooler.